myapp/playlist.js

```diff
@@ -0,0 +1,67 @@
+var songJSON=JSON.parse(playlistsJSON);
+var l=songJSON.length;
+
+var playlist=[];
+var songs=[];
+
+
+
+function Playlist(data){
+  this.id=data.id;
+  this.title=data.title;
+  this.songs=[];
+  for(j=0;j<data.songs.length;j++)
+  {
+    var tempSong = new Song(data.songs[j]);
+    this.songs.push(tempSong);
+  }
+}
+
+for(i=0;i<l;i++){
+  playlist[i]=new Playlist(songJSON[i]);
+  // console.log(playlist[i]);
+  // console.log(playlist[i].songs[0]);
+}
+
+
+function removePlaylist(){
+  var parent = document.getElementById( 'plyL' );
+  while ( document.getElementById('song') != null ){
+    var child = document.getElementById( 'song' );
+    parent.removeChild( child );
+  }
+}
+
+var lengthSongs = songJSON[0].songs.length;
+function displaySongs(){
+    removePlaylist();
+    var playlistElement = document.getElementById('plyL');
```

Comment:
Naming is terrible. Assign very clear and very well names to:
1. variables
2. css identifiers: id, class names